pugdebug/message_parser.py

```diff
@@ -137,20 +138,36 @@ def parse_breakpoint_list_message(self, message):
 
         return breakpoints
 
+    def parse_eval_message(self, message):
+        xml = xml_parser.fromstring(message)
+        child = xml[0]
+
+        # Detect errors as having an <error> child
+        if child.tag == '{urn:debugger_protocol_v1}error':
```

Comment:
I find `if child.tag.endswith('error')` better if in the future the urn changes.